akka-remote/src/test/scala/akka/remote/artery/ArterySmokeTest.scala

```diff
@@ -11,7 +11,11 @@ object ArterySmokeTest {
   val commonConfig = """
      akka {
        actor.provider = "akka.remote.RemoteActorRefProvider"
-       remote.artery.enabled = on
+       remote.artery {
+         enabled = on
+         #transport = tcp
```

Comment:
do we really want to make this configurable? I only used TCP because it was the one readily available.